templates/taxi/manufacturer_list.html

```diff
@@ -1,28 +1,35 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
+    <h1>Manufacturer List <a href="{% url 'taxi:manufacturer-create' %}">+</a></h1>
 
     {% if manufacturer_list %}
       <table class="table">
         <tr>
           <th>ID</th>
           <th>Name</th>
           <th>Country</th>
+					<th>Update</th>
+					<th>Delete</th>
         </tr>
 
         {% for manufacturer in manufacturer_list %}
             <tr>
               <td>
-                  {{ manufacturer.id }}
+								<a href="{% url "taxi:car-detail" pk=manufacturer.id %} ">{{ manufacturer.id }}</a>
```

Comment:
The URL tag for the manufacturer detail view seems to be incorrect. It currently points to 'taxi:car-detail' instead of a manufacturer detail view. Ensure that the correct URL name for the manufacturer detail view is used here.